sources/osgAnimation/BasicAnimationManager.js

```diff
@@ -135,19 +130,12 @@ define( [
             // animations to start
             this._startAnimations = {};
 
-            this._targets.length = 0;
-            this._targetsMap = {};
-
-            this._targetID.length = 0;
-
-            var i;
-            for ( i = 0; i < this._targetIDByTypes.length; i++ )
-                this._targetIDByTypes[ i ].length = 0;
+            this._resetTargets();
 
             this._activeAnimations = {};
             this._activeAnimationList.length = 0;
 
-            for ( i = 0; i < this._activeChannelsByTypes; i++ )
+            for ( var i = 0; i < this._activeChannelsByTypes; i++ )
```

Comment:
not sure understand what do you mean ? reset array should be lenght = 0, but we keep some predefined array